tensorflow/core/api_def/base_api/api_def_Diag.pbtxt

```diff
@@ -18,12 +18,14 @@ rank 2k with dimensions [D1,..., Dk, D1,..., Dk] where:
 
 For example:
 
-```
-# 'diagonal' is [1, 2, 3, 4]
-tf.diag(diagonal) ==> [[1, 0, 0, 0]
-                       [0, 2, 0, 0]
-                       [0, 0, 3, 0]
-                       [0, 0, 0, 4]]
-```
+"""
```

Comment:
Don't put the triple quotes in here as this exits the docstring and it's not valid python syntax

Reply:
Do u mean I should make like this?
`""" >>> diagonal = [1, 2, 3, 4]`